pom.xml

```diff
@@ -531,7 +531,7 @@
 			<dependency>
 			  <groupId>joda-time</groupId>
 			  <artifactId>joda-time</artifactId>
-			  <version>2.10.5</version>
+			  <version>2.10.6</version>
```

Comment:
try out the changes with reference application

Reply:
hello @dkayiwa i tested with the standalone reference application application :  
1.I use the command mvn clean install, I obtain a war file,
2. I replace the war file of the standalone reference application by my war file which I obtained
3.and I use the command java -jar openmrs-standalone.jar to run the standalone reference application with my war that I replaced.
and this  is the output : 
![Screenshot from 2020-06-16 17-51-03](https://user-images.githubusercontent.com/39851246/84805072-99310480-affb-11ea-94f9-4ba66d6b5d50.png)